CODEOWNERS

```diff
@@ -1,5 +1,5 @@
 # Protocol owners are not generally differentiated by sub-area for simplicity
-/** @jordanschalm @AlexHentschel @durkmurder @zhangchiqing @peterargue @Kay-Zee @janezpodhostnik @tarakby
+/** @onflow/flow-core-protocol
```

Comment:
Says this is a syntax error because the team doesn't have write access to the repo. Probably the users do individually, but not the team? Not sure. 

<img width="1054" alt="image" src="https://github.com/user-attachments/assets/afadb904-6755-410e-96b9-03ea169c7f00" />

Reply:
Added them now - where do you get that notice, couldn't find it.

Reply:
It was just inline in the PR Files Changed tab -- gone now though.